client/deis.py

```diff
@@ -299,7 +299,7 @@ def readable_datetime(datetime_str):
     now = datetime.now(timezone)
     delta = relativedelta.relativedelta(now, dt)
     # if it happened today, say "2 hours and 1 minute ago"
-    if delta.days <= 1 and dt.day == now.day:
+    if dt > (now - relativedelta.relativedelta(days=1)):
```

Comment:
we should probably move line 321 (the line where we initiate `yesterday`) above this line such that there's no duplicate code here